src/Console/IdeHelperCommand.php

```diff
@@ -108,6 +111,14 @@ protected function buildSchemaString(array $directiveClasses): string
                 .$definition."\n";
         }
 
+        // append programmatically registered types
+        $typeRegistry = resolve(TypeRegistry::class);
```

Comment:
```suggestion
        $typeRegistry = app(TypeRegistry::class);
```

This removes one layer of indirection. Injection through `handle()` would be even better though, it is good to see the dependencies upfront.